units/crons_remote_hour.go

```diff
@@ -46,7 +46,7 @@ func (j *cronsRemoteHourJob) Run(ctx context.Context) {
 	}
 
 	ops := []amboy.QueueOperation{
-		PopulateCacheHistoricalTestDataJob(2),
+		PopulateCacheHistoricalTaskDataJob(2),
```

Comment:
now that it's not calculating hourly stats maybe we only need it to run once a day?

Reply:
I am nervous that this would end up being a lot of work for the database at once if we do it this way.